lib/mix/tasks/nerves_hub.device.ex

```diff
@@ -67,6 +67,8 @@ defmodule Mix.Tasks.NervesHub.Device do
     cert: :string
   ]
 
+  @data_path Path.join(File.cwd!(), "nerves-hub")
```

Comment:
Did you want this to be evaluated at compile-time or runtime?

Reply:
Good call, no I dont